docs/modules/integrate/pages/cdc-connectors.adoc

```diff
@@ -79,20 +187,28 @@ For example, a sink mapping CDC data to a `Customer` class and
 maintaining a map view of latest known email addresses per customer
 (identified by ID) would look like this:
 
-```java
+[source,java]
+----
 Pipeline p = Pipeline.create();
 p.readFrom(source)
  .withoutTimestamps()
  .writeTo(CdcSinks.map("customers",
     r -> r.key().toMap().get("id"),
     r -> r.value().toObject(Customer.class).email));
-```
+----
 
 [NOTE]
 ====
 The key and value functions have certain limitations. They can be used to map only to objects which the Hazelcast member can deserialize, which unfortunately doesn't include user code submitted as a part of the job. So in the above example it's OK to have `String` email values, but we wouldn't be able to use `Customer` directly.
 
 If user code has to be used, then the problem can be solved with the help of the User Code Deployment feature. Example configs for that can be seen in our xref:pipelines:cdc-join.adoc#7-start-hazelcast-jet[CDC Join tutorial].
+====
+
+== Migration Guide
```

Comment:
Is this a guide or just tips?
If you were writing this a step by step guide, even high level steps, I would keep the Guide nomenclature but otherwise I would consider renaming this section i.e. I think we are overselling it as a guide.

Reply:
idea was to provide a guide, but TBH during writing of it I realised that it should be so simple, that detailed guide is not needed. Just needed to update the title, which I forgot ;)